### DISCUSSION TITLE
More user-friendly Chinese translation

### DISCUSSION THREAD

**Discussion by Zhou-Shilin** [community/community#49771] [Mobile] [answered]
The Simplified Chinese translation of Github mobile version is not user-friendly. When someone who is used to using Github's web version switches to the mobile version, he or she will be very uncomfortable. In fact, **in Github Chinese community we refer to use “Issues” as issues rather than "议题 (topic)"**. For example, we always say "如果你遇到了Bug或者是想要添加新功能，请在Github提交一个Issue)" or "如果有能力的话，大佬们也可以Pull一个request".
	For words like Issues, pull request, etc., Chinese Githubers prefer to use English directly instead of "议题" or "拉取请求". Because the word 议题 (topic) is different from "issue", it has a special meaning in this context, which cannot be expressed by "议题". Therefore, there is no Chinese equivalent for the word "issue" (there is no such concept in Chinese). The word "issue" itself has already become "Chinese". Just like the English translation of "粽子", which is Zongzi, it become a _foreign word_.
	I hope the Chinese translation of Github mobile version is more user-friendly and in line with user habits. I suggest using the following words directly in English instead of forcing a direct translation:
issue(s)
pull request(s)
star(s)
release(s)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, 版本 1.103.0 (93212509)
Device: iPad Air 5 (iOS 16.2)
</details>

**Comment by aconsuegra** [comment, marked as answer]
Thanks a lot for your feedback @Zhou-Shilin! We are already not translating GitHub specific terms like issues or pull requests in some languages, like Japanese, so we can do the same for Simplified Chinese. I'm going to create an internal issue to track this request.
